tigerpath/majors_and_certificates/majors/ANT.json

```diff
@@ -15,14 +15,9 @@
       "email": "crouse@princeton.edu"
     },
     {
-      "type": "Acting Chair",
-      "name": "Serguei A. Oushakine",
-      "email": "oushakin@princeton.edu"
-    },
-    {
-      "type": "Director of Undergraduate Studies",
-      "name": "Ryo Morimoto",
-      "email": "ryo.morimoto@princeton.edu"
+      "type": " Director of Undergraduate Studies",

```

Comment:
This might seem kind of nitpicky but there's an extra space at the beginning here 😬

Can you fix it? 🙏 Thanks!

Reply:
Fixed!